migrations/env.py

```diff
@@ -17,7 +17,12 @@
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
-sqlalchemy_url = str(engine.url) if engine else Config.SQLALCHEMY_DATABASE_URI
+
+if engine:
+    sqlalchemy_url = engine.url.render_as_string(hide_password=False)
```

Comment:
This fixes the test failures on the `main` branch. (I could put this in a separate PR, but this one's small anyway.)

SQLAlchemy has [recently](https://pypi.org/project/SQLAlchemy/#history) (in October) released beta versions of their new 2.0 version coming out. Our app is picking up those beta versions because we only have a version floor, and it's a beta version:

https://github.com/voltrondata-labs/arrow-benchmarks-ci/blob/102e9943829b8afb2a1b289d538250273e7c559c/requirements.txt#L10

[One of the changes](https://github.com/sqlalchemy/sqlalchemy/pull/8563) in 2.0 is that `str(engine.url)` now redacts the password, substituting `***` for the actual password. This was causing alembic to not be able to connect, because we were literally supplying "***" as the password instead of "postgres". 

While the solution is simple, this was probably one of the most difficult things for me to debug in a while. 😄 I finally got it by printing out `sqlalchemy_url` here and checking the docker-compose logs.

Reply:
Great work! Thank you for the explanation!
Note need for a separate PR.